package.json

```diff
@@ -54,7 +54,7 @@
   "dependencies": {
     "bitcore": "^0.12.0",
     "bloom-filter": "^0.2.0",
-    "buffers": "^0.1.1",
+    "buffers": "olalonde/node-buffers",
```

Comment:
The changes to node-buffers look good, however this references an unpublished master branch. There are a few open issues, https://github.com/bitpay/bitcore-p2p/issues/52 and https://github.com/bitpay/bitcore-p2p/issues/32 with this already. I believe we only use a small portion of the implemented methods, so we may be able to make this a part of this codebase, since we're also adding the skip method. However we can take care of this in another PR as well as the deprecation warnings.